flask5/hotel/resources.py

```diff
@@ -0,0 +1,157 @@
+from flask_restful import Resource, marshal_with, abort
+from sqlalchemy.exc import IntegrityError
+from .parsers import *
+from .fields_formatting import *
+from .fields_check import *
+from .model import *
+from .db import db
+
+
+class Rooms(Resource):
+    @marshal_with(rooms_structure)
+    def get(self):
+        args = rooms_parser.parse_args(strict=True)
+        if not any(args.values()):
+            return RoomsModel.query.all()
+        n_args = {k: v for k, v in args.items() if v}
+        return RoomsModel.query.filter_by(**n_args).all()
+
+    def post(self):
+        args = rooms_parser.parse_args(strict=True)
+        check_if_all_parameters_is_filled(args)
+        try:
+            db.session.add(RoomsModel(**args))
+            db.session.commit()
+        except IntegrityError:
+            abort(409, error_code=409,
+                  message=f'Room {args["number"]} is already exists!')
+        return f'New room {args["number"]} was successfully added'

```

Comment:
return all room fields.
client should know what room with what fields was created